src/lib/mapping.js

```diff
@@ -70,7 +70,8 @@ function mapTeamList (row) {
     description: row.description,
     path: row.path,
     organizationId: row.org_id,
-    usersCount: parseInt(row.users_count)
+    usersCount: parseInt(row.users_count),
```

Comment:
Well it defaults to 10 anyway. I thought the use default was deprecated though but if node doesn't complain we're properly using the function as per its specs.